CODEOWNERS

```diff
@@ -6,85 +6,81 @@
 
 # Order is important. The last matching pattern has the most precedence.
 
-# Recovery
-modules/bitgo/src/v2/recovery.ts @BitGo/sdk-team
-
 # UTXO coin modules
+modules/abstract-utxo/ @BitGo/btc-team
 modules/bitgo/test/v2/unit/coins/utxo/ @BitGo/btc-team
 modules/bitgo/test/v2/unit/lightning.ts @BitGo/btc-team
-modules/abstract-utxo/src/ @BitGo/btc-team
-modules/sdk-coin-bch/src/ @BitGo/btc-team
-modules/sdk-coin-bch/test/ @BitGo/btc-team
-modules/sdk-coin-bcha/src/ @BitGo/btc-team
-modules/sdk-coin-bcha/test/ @BitGo/btc-team
-modules/sdk-coin-bsv/src/ @BitGo/btc-team
-modules/sdk-coin-bsv/test/ @BitGo/btc-team
-modules/sdk-coin-btc/src/ @BitGo/btc-team
-modules/sdk-coin-btc/test/ @BitGo/btc-team
-modules/sdk-coin-btg/src/ @BitGo/btc-team
-modules/sdk-coin-btg/test/ @BitGo/btc-team
-modules/sdk-coin-dash/src/ @BitGo/btc-team
-modules/sdk-coin-dash/test/ @BitGo/btc-team
-modules/sdk-coin-ltc/src/ @BitGo/btc-team
-modules/sdk-coin-ltc/test/ @BitGo/btc-team
-modules/sdk-coin-zec/src/ @BitGo/btc-team
-modules/sdk-coin-zec/test/ @BitGo/btc-team
+modules/blockapis/ @BitGo/btc-team
+modules/bls-dkg/ @BitGo/hsm
```

Comment:
You'll need to provide the HSM Team write access to the SDK repo due to this change. Please reach out if you need help on how to do this.

Reply:
Thanks. I've gone ahead and created the ticket and PR: https://bitgoinc.atlassian.net/browse/BG-56932